LKMEX-whitelist/combased-lkmex.md

```diff
@@ -0,0 +1,43 @@
+## Name of the project
+Comverse
+
+## Description of the project
+Combased project is web3 agency and also a leader in e-commerce space. Read more here https://combased.io
+
+Also check our previous PR:
+https://github.com/ElrondNetwork/elrond-specs/pull/29
+
+## Purpose
+We send to our holders monthly rewards with LKMEX, UTK, RIDE and EGLD.
+
+To farm LKMEX on the dex we use our CDO wallet, which takes 50% of all royalties and earnings from our services. We provide web3 development services and sell our Shopify apps.
+
+To qualify all eligible addresses we use our first open source script, which was actually instrumental in developing Elrond ecosystem.
+
+https://github.com/Combased/elrond-lkmex-distribution
+
+
+## Address of Smart Contract
+erd1qqqqqqqqqqqqqpgq9esqwrwuk7fsdums5nz2rczqweu0lyudmkdqudmg2r
+
+Deployed on mainnet
+
+## Smart Contract Code
+https://github.com/xdevguild/sc-multi-sender-rs
```

Comment:
This contract is too general. It can be used for secondary markets so we cannot accept this. You need a contract where it is clear that it is used for NFT staking.

Reply:
What do you mean with NFT staking? As I said we won't allow sending NFTs to some unknown SC, and keeping them. Please show me approved example which DOES NOT send NFTs and is still approved, I am wasting my time and yours, without having clear indications. And response time is horrible...

Reply:
They never asked me to use staking in order to distribute the XMEX, this is what they told me first time they rejected me:

> We do not whitelist user addresses, only smart contracts. Distribution of tokens can happen through SCs. You can collaborate with other projects who have smart contracts and help you distribute through those.

And now they just changed the condition, without telling us !

Reply:
Yes, @sasurobert please stop wasting our time and yours as well. We as Combased focus on real e-commerce projects. Actually we even contacted your team about mutual collab, so it would be wiser to actually think about driving next wave of blockchain based e-commerce innovations instead of spending time on simple SCs.

Reply:
@sasurobert we need your answer asap.

Reply:
ahaha ASAP, @sasurobert just don't give a f** about us, better promote rug pulls on [xspotlight](https://xspotlight.com/) 🤣

Reply:
We did not change conditions, and I do not waste my time and not even yours. Only smart contracts which cannot be used for secondary markets are accepted. The contract defined above can be easily integrated in a lockedMexExchange like contract and that would start the secondary market for xMEX, which we clearly do not want.
Please update the code for your contract. We cannot whitelist it.